addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -2008,3 +2008,42 @@ def test_in_invoice_line_tax_line_delete(self):
 
         # Assert the tax line is recreated for the tax
         self.assertIn(tax, self.invoice.line_ids.tax_line_id)
+
+        tax0 = self.env['account.tax'].create({
```

Comment:
This is a different test, it doesn't depend at all on the previous one.
Please make a distinct function for it